backends/cuda-gen/ceed-cuda-gen-operator-build.cpp

```diff
@@ -1106,7 +1106,7 @@ extern "C" int CeedCudaGenOperatorBuild(CeedOperator op) {
         break;
       case CEED_EVAL_GRAD:
         code << "  CeedScalar r_q"<<i<<"[ncomp_in_"<<i<<"*Dim];\n";
-        //TODO put the following in colograd3d
+        //TODO put the following in collograd3d
```

Comment:
Is this still TODO or the comment is outdated?

Reply:
Done.